docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,76 @@
    "id": "840ea662",
```

Comment:
<p>Title -</p><p>Observe the Check's Output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>